T/TempestRemap/build_tarballs.jl

```diff
@@ -72,13 +69,13 @@ products = [
 ]
 
 dependencies = [
-    Dependency("libblastrampoline_jll"),
+    Dependency("MKL_jll"),
     Dependency("NetCDF_jll"),
     Dependency("HDF5_jll"),
     # The following is adapted from NetCDF_jll
     BuildDependency(PackageSpec(; name="MbedTLS_jll", version=v"2.24.0")),
 ]
 
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-    julia_compat="1.7",
+    julia_compat="1.6",
```

Comment:
I don't recommend changing the compat bound. This change may make registration to fail (not sure), and in case you want to come back to libblastrampoline later you'll to switch back to 1.7 again

Reply:
ok makes sense